channel_log.go

```diff
@@ -34,6 +35,29 @@ func NewChannelError(message, code string) *ChannelError {
 	return &ChannelError{message: message, code: code}
 }
 
+func ErrorResponseStatusCode() *ChannelError {
+	return NewChannelError("Unexpected response status code.", "core:response_status_code")
```

Comment:
don't need to repeat the status code in the error message because they can see it in the trace above